src/components/layout/navbar-profile-menu.tsx

```diff
@@ -1,60 +1,149 @@
 'use client'
 
 import { AppShellSection, Collapse, NavLink } from '@mantine/core'
+import { useDisclosure, useClickOutside, useHotkeys } from '@mantine/hooks'
+import { CaretRight, SignOut, User } from '@phosphor-icons/react/dist/ssr'
+import { useClerk } from '@clerk/nextjs'
 import { UserAvatar } from '@/components/user-avatar'
 import { UserName } from '@/components/user-name'
-import { CaretRight, SignOut, User } from '@phosphor-icons/react/dist/ssr'
+import { useRef } from 'react'
 import styles from './navbar-items.module.css'
-import { useDisclosure } from '@mantine/hooks'
-import { useClerk } from '@clerk/nextjs'
 
 export function NavbarProfileMenu() {
     const { signOut, openUserProfile } = useClerk()
-    const [opened, { toggle }] = useDisclosure()
+    const [opened, { toggle, close }] = useDisclosure()
+
+    const toggleButtonRef = useRef<HTMLAnchorElement>(null)
+    const accountMenuItemRef = useRef<HTMLAnchorElement>(null)
+    // const reviewerKeyMenuItemRef = useRef<HTMLAnchorElement>(null)
```

Comment:
I was confused first. But it seems it's referring to the the future lines 102-113.

I know it sounds silly but maybe add a (TODO) comment on why this commented code is here.